lib/SoftwareUserStatus.py

```diff
@@ -100,6 +101,33 @@ def refetchUserStatusLazily(tryCountUntilFoundUser):
 
         updateSessionSummaryFromServer(True)
 
+def authCompletionHandler(user):
```

Comment:
Does this get called anywhere?

Reply:
oh that. that was to prepare for the websocket logic. I had websocket-client and that kept failing due to not finding another module. I decided to take out the websocket logic until I figure that out

Reply:
i removed that method for now